src/components/projekti/suunnitteluvaihe/komponentit/VuorovaikutustilaisuusDialog/index.tsx

```diff
@@ -239,8 +239,9 @@ export default function VuorovaikutusDialog({
                 <p>Voit valita saman vuorovaikutustavan useammin kuin yhden kerran.</p>
```

Comment:
Ja eikös tämä ole väärin päin tässä, vaikka en ihan varma ole tuon mostlyDisabledin käytöstä, mutta ihan ensimmäisellä kerralla täytettäessä taitaa ainakin olla undefined ja tulee tuo julkaisun jälkeinen teksti...